gapic-generator-java/src/test/resources/echo_without_mixins_v1beta1.yaml

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+type: google.api.Service
+config_version: 3
+name: showcase.googleapis.com
+title: Client Libraries Showcase API
+
+apis:
```

Comment:
Per https://google.aip.dev/client-libraries/4234, mixins must be configured in `apis` section.
```
To be generated into a client library, a mixin service must be declared under [apis](https://github.com/googleapis/googleapis/blob/master/google/api/service.proto#L96) in a [google.api.Service](https://github.com/googleapis/googleapis/blob/master/google/api/service.proto). 
Furthermore, only the RPCs with [google.api.http](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto) bindings declared in the [http](https://github.com/googleapis/googleapis/blob/master/google/api/service.proto#L124) configuration of the same [google.api.Service](https://github.com/googleapis/googleapis/blob/master/google/api/service.proto) can be generated (more details on these bindings in [AIP-127](https://google.aip.dev/127)). If a mixin service RPC does not have a [google.api.http](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto) rule declared in the [google.api.Service](https://github.com/googleapis/googleapis/blob/master/google/api/service.proto), it must not be generated.
```

I think we should have at least 4 test cases:
1. Mixins should be generated if they are configured in both `apis` and `http` sections.
2. Mixins should not be generated if they are configured in `apis` but not in `http` sections.
3. Mixins should not be generated if they are configured in `http` but not in `apis` sections. This is probably a misconfiguration.
4. Mixins should not be generated if they are not configured in `apis` or `http` sections.

Reply:
Thanks for pointing it out.

I'll add two tests for each scenario, one with Mixin protos and one without.